postgres_admin_panel/tasks/list_active_connections.sql

```diff
@@ -0,0 +1,13 @@
+select 
+  client_addr, 
+  usename, 
+  datname, 
+  count(*) 
+from 
+  pg_stat_activity 
+group by 
+  1, 
+  2, 
+  3 
+order by 
+  4 desc;
```

Comment:
can you run pgformatter on the SQL files?
```suggestion
SELECT
  client_addr,
  usename,
  datname,
  count(*)
FROM
  pg_stat_activity
GROUP BY
  1,
  2,
  3
ORDER BY
  4 DESC;

```